docs/mobile-apps/faq.md

```diff
@@ -96,7 +96,7 @@ Yes. We support OAuth login via Google and GitHub.
 
 #### **Are your devices jailbroken?**
 
-No. We do not offer jailbroken or rooted devices.
+No. We do not offer jailbroken or rooted devices, or penetration testing. 
```

Comment:
@kristofmuhi 

I'm assuming you mean we don't support "penetration testing", if so, can you change this sentence a bit/remove this and place it somewhere else? It's a bit out of context now

Reply:
reverted